namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-prod/05-certificate.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: checkmydiary.service.justice.gov.uk
+  namespace: checkmydiary-prod
+spec:
+  secretName: checkmydiary-cert
```

Comment:
secretName must match the one the Ingress expects:
`$ kubectl --context live-1 -n check-my-diary-prod get ing check-my-diary-prod-ingress -o json | jq -r '.spec.tls'
[
  {
    "hosts": [
      "checkmydiary.service.justice.gov.uk"
    ],
    "secretName": "tls-certificate"
  }
]`